openapi/paths/{resource}@{resourceId}@external-identifiers@{service}.yaml

```
@@ -0,0 +1,108 @@
+  - $ref: ../components/parameters/externalIdentifierResourceId.yaml
+  - $ref: ../components/parameters/externalIdentifierService.yaml
+get:
+  x-products:
+    - Users
+  tags:
+    - External identifiers
+  summary: Retrieve an external identifier
+  operationId: GetExternalIdentifier
+  x-sdk-operation-name: get
+  description: Retrieves an external identifier with a specified resource, resource ID and a service name.
+  responses:
+    '200':
+      description: External identifier retrieved.
+      content:
+        application/json:
+          schema:
+            $ref: ../components/schemas/ExternalIdentifier.yaml
+    '401':
+      $ref: ../components/responses/Unauthorized.yaml
+    '403':
+      $ref: ../components/responses/Forbidden.yaml
+    '404':
+      $ref: ../components/responses/NotFound.yaml
+post:
+  x-products:
+    - Users
+  tags:
+    - External identifiers
+  summary: Sync an external identifier
+  operationId: PostExternalIdentifier
+  x-sdk-operation-name: sync
+  description: |-
+    Sync an external identifier with a specified resource, resource ID and a service name.
+  responses:
+    '204':
+      description: External identifier sync scheduled.
+    '401':
+      $ref: ../components/responses/Unauthorized.yaml
+    '403':
+      $ref: ../components/responses/Forbidden.yaml
+    '404':
+      $ref: ../components/responses/NotFound.yaml
+    '422':
+      $ref: ../components/responses/ValidationError.yaml
+put:
+  x-products:
+    - Users
+  tags:
+    - External identifiers
+  summary: Upsert an external identifier
+  operationId: PutExternalIdentifier
+  x-sdk-operation-name: update
+  description: |-
+    Creates or updates an external identifier with a specified resource, resource ID and a service name.
+  requestBody:
+    content:
+      application/json:
+        schema:
+          $ref: ../components/schemas/ExternalIdentifier.yaml
+    required: true
+  responses:
+    '200':
+      description: External identifier updated.
+      content:
+        application/json:
+          schema:
+            $ref: ../components/schemas/ExternalIdentifier.yaml
+    '201':
+      description: External identifier created.
+      headers:
+        Location:
+          $ref: ../components/headers/Location.yaml
+      content:
+        application/json:
+          schema:
+            $ref: ../components/schemas/ExternalIdentifier.yaml
+    '401':
+      $ref: ../components/responses/Unauthorized.yaml
+    '403':
+      $ref: ../components/responses/Forbidden.yaml
+    '404':
+      $ref: ../components/responses/NotFound.yaml
+    '422':
+      $ref: ../components/responses/ValidationError.yaml
+delete:
+  x-products:
+    - Users
+  tags:
+    - External identifiers
+  summary: Delete an external identifier
+  operationId: DeleteExternalIdentifier
+  x-sdk-operation-name: delete
+  description: |-
+    Deletes an external identifier with a specified resource, resource ID and a service name.
```

nitpick: [serial comma](https://product.rebilly.dev/writing-style-guide/grammar-and-mechanics/commas/#serial-commas)

We also shouldn't add block scalers for single line descriptions ([line break guidance](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#line-breaks))

```suggestion
  description: Deletes an external identifier with a specified resource, resource ID, and service name.